src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx

```diff
@@ -11,6 +11,11 @@ export interface AboutPartnerProps {
 export const AboutPartner: React.FC<AboutPartnerProps> = ({
   partner: { profile, vatNumber, website, displayFullPartnerPage },
 }) => {
+  const tracking = useTracking()
+  const tappedViewTrackingData = {
+    // TODO: add tracking properties
```

Comment:
@louislecluse / @bbaiyr (not sure who the request should go to, since Louis and I had talked about this on Slack but Baiyr is now FX data person 😅) Would one of you mind defining tracking properties for this in Cohesion?